caps/capabilities.json

```diff
@@ -23,7 +23,7 @@
   },
   "hostMachines": [
     {
-      "machineIP": "127.0.0.1",
+      "machineIP": ["127.0.0.1"],
```

Comment:
@ShamaUgale What if there are different xcode versions with different machines. So we should ideally accept both formats string and array.

Case 1: Diff sim's or diff xcode versions on diff machines
Case 2: lot of machines (ips) with same sim and xcode versions.

Reply:
@SrinivasanTarget wrt to "case 1", are you also referring to OS being a list/string?

Reply:
So this PR covers Case2. But will break for Case 1. As srini mentioned we need to support both String and Array for machineIP

Reply:
i am accommodating it.